src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -215,5 +215,7 @@ public static class ScriptConstants
 
         public static readonly string FunctionMetadataDirectTypeKey = "DirectType";
         public static readonly string LiveLogsSessionAIKey = "#AzFuncLiveLogsSessionId";
+
+        public static readonly string ScaleControllerFeatureFlags = "ScaleControllerFeatureFlags";
```

Comment:
This can be removed, as the format of ScmHostingConfiguration should be `feature1=1,eventhubtargetscaler=1,servicebustargetscaler=1`, etc. There is no longer feature called `ScaleControllerFeatureFlags`